doc/modules/calibration.rst

```diff
@@ -11,16 +11,21 @@ When performing classification you often want not only to predict the class
 label, but also obtain a probability of the respective label. This probability
 gives you some kind of confidence on the prediction. Some models can give you
 poor estimates of the class probabilities and some even do not support
-probability prediction. The calibration module allows you to better calibrate
+probability prediction (e.g., :class:`SGDClassifier`). The calibration
```

Comment:
you need the whole path unless there's a previous sphinx directive indicating the current module (which wouldn't be sklearn.linear_model anyway)

```suggestion
probability prediction (e.g., :class:`~sklearn.linear_model.SGDClassifier`). The calibration
```

Reply:
ooohhhh thanks, that is good to know.